tox.ini

```diff
@@ -41,37 +40,41 @@ commands = isort --profile black --check --diff .
 description = Code formatting with Black and isort (and possibly other tools in the future)
 basepython = python3
 deps =
-          {[testenv:black]deps}
-          {[testenv:isort]deps}
```

Comment:
are these indentation issues on purpose or the result of some kind of autoformatter?

Reply:
They're on purpose. The file's indentation was sort of messed up (a combination of 4 and 5 space indents and tabs) so I went ahead and changed it.